.github/workflows/nightly_macos_x86_64.yml

```diff
@@ -14,16 +14,15 @@ jobs:
           
       - name: write version to file
         run: ./ci/write_version.sh
-        
-      # build has to be done before tests #2572
```

Comment:
This is no longer an issue in this particular case but I'm not sure if we can close #2572